### PR TITLE
enable EPIC mongo backups using the bcdevops backup container

### PR DESCRIPTION
**Pull request by mtCarto**
[EE-1086](https://bcmines.atlassian.net/browse/EE-1086)

- adds configuration for backups of EPIC's prod mongodb
- updates the naming of the metabase postgres backup objects for improved clarity

### CODE REVIEW

**Review by marklise — APPROVED** [review]

File: openshift/templates/db-backup/mongo/db-backup.dc.json [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+      "description": "The name of the image to use for this resource.  Use 'backup-{database name}' depending on your database provider",
+      "required": true,
+      "value": "backup-mongodb"
+    },
+    {
+      "name": "IMAGE_NAMESPACE",
+      "displayName": "Image Namespace",
+      "description": "The namespace of the OpenShift project containing the imagestream for the application.",
+      "required": true,
+      "value": "backup-container"
```
Comment (MaxWardle) [inline openshift/templates/db-backup/mongo/db-backup.dc.json]: Should this be esm?

File: openshift/templates/db-backup/mongo/db-backup.dc.json [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+      "description": "The database password key name stored in database deployment resources specified by POSTGRES_DEPLOYMENT_NAME.",
+      "required": true,
+      "value": "database-password"
+    },
+    {
+      "name": "MONGO_DEPLOYMENT_NAME",
+      "displayName": "Database Deployment Name",
+      "description": "The name associated to the database deployment resources.  In particular, this is used to wire up the credentials associated to the database.",
+      "required": true,
+      "value": "mongo"
```
Comment (MaxWardle) [inline openshift/templates/db-backup/mongo/db-backup.dc.json:364]: unsure of this too

File: openshift/templates/db-backup/mongo/db-backup.dc.json [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+      "description": "FTP password",
+      "required": false,
+      "value": ""
+    },
+    {
+      "name": "WEBHOOK_URL",
+      "displayName": "Webhook URL",
+      "description": "The URL of the webhook to use for notifications.  If not specified, the webhook integration feature is disabled.",
+      "required": false,
+      "value": ""
```
Comment (MaxWardle) [inline openshift/templates/db-backup/mongo/db-backup.dc.json:433]: We should probably set this up too

Comment (mtCarto) [reply openshift/templates/db-backup/mongo/db-backup.dc.json:433]: will make it a secret ref

File: openshift/templates/db-backup/mongo/db-backup.dc.json (hunk 2 above)
Comment (mtCarto) [reply openshift/templates/db-backup/mongo/db-backup.dc.json:364]: will keep in params file

File: openshift/templates/db-backup/mongo/db-backup.dc.json (hunk 1 above)
Comment (mtCarto) [reply openshift/templates/db-backup/mongo/db-backup.dc.json]: will switch this, was defined by params file

File: openshift/templates/db-backup/mongo/db-backup.dc.json (hunk 3 above)
Comment (mtCarto) [reply openshift/templates/db-backup/mongo/db-backup.dc.json:433]: template already has this url setup as a secret

**Review by MaxWardle — APPROVED** [review]

**Pull request merged**